testapp/.meteor/release

```diff
@@ -1 +1 @@
-METEOR@2.14
```

Comment:
The test app should be a 3.0 application, no?

Reply:
I was thinking about this and in the end settled on 2.16. In theory the test suite matrix should dictate this version, but I don't think that is the case. Since we are still backward compatible to 2.8.1 maybe the best approach would be to create another test suite that is v3 app. Thoughts?

Reply:
Yeah, I think having a dedicated 3.0 test application is the way to go, though it's going to be cumbersome maintaining two test applications at the same time, I'd recommend to double down on 3.0 but it's up to you if you can dedicate the time.  :+1:

Reply:
Well, there are some other things in motion which do not make it a big issue. Just need to get it working properly.

Reply:
If you want a test matrix in GH Actions then you need to create a new Meteor bare project each time the matrix release changes so we need to adjust the CI and use the test project only locally.

I can provide that if you need.

Reply:
Maybe later. I don't think that the effort is worth it at the moment, given the future plans for this package. The current setup is good enough.